### PR TITLE
Added Php code sniffer to the project

### PR DESCRIPTION
**Pull request by oldDriver**
Code style check with the PHP Code Sniffer tool, which validate code with a coding standard

### CODE REVIEW

File: tests/unit/.gitingore [hunk 1]
```diff
@@ -1,2 +0,0 @@
-.DS_Store
```
Comment (mccottry) [inline tests/unit/.gitingore]: This is an unrelated change.  Remove it from this PR.

Comment (oldDriver) [reply tests/unit/.gitingore]: @mccottry it is typo "gitingor" instead of "gitignore"
May be it make sense stay it here?

Comment (mccottry) [reply tests/unit/.gitingore]: Open another PR for this.  And create the appropriate task and subtask in Asana.
